skyportal/facility_apis/ztf.py

```diff
@@ -20,22 +28,26 @@
 else:
     ZTF_URL = f"{cfg['app.ztf.protocol']}://{cfg['app.ztf.host']}:{cfg['app.ztf.port']}"
 
+ZTF_FORCED_URL = cfg['app.ztf_forced_endpoint']
+
 bands = {'g': 1, 'ztfg': 1, 'r': 1, 'ztfr': 2, 'i': 3, 'ztfi': 3}
 inv_bands = {v: k for k, v in bands.items()}
```

Comment:
I'm not sure how you are inverting the dictionary when there are identical values.

Reply:
very good point.